app/src/components/media/item/RowItem.tsx

```diff
@@ -0,0 +1,80 @@
+import { Grid, Image, Text, useTheme } from '@nextui-org/react';
+import { Link } from '@remix-run/react';
+import useMediaQuery from '~/hooks/useMediaQuery';
+import { IMedia } from '~/services/tmdb/tmdb.types';
+
+interface IRowItem {
+  item: IMedia;
+}
+const RowItem = ({ item }: IRowItem) => {
+  const isXs = useMediaQuery(425, 'max');
+  const { isDark } = useTheme();
+
+  const href = (item.mediaType === 'movie' ? '/movies/' : '/tv-shows/') + item.id;
+  const year = item.releaseDate ? new Date(item.releaseDate).getFullYear() : undefined;
+  const overview = item.overview && isXs ? `${item.overview.substring(0, 100)}...` : item.overview;
+
+  return (
+    <Grid.Container
+      gap={1}
+      alignItems="flex-start"
+      css={{
```

Comment:
@ctuanle đây t nghĩ thêm maxWidth: '1400px' đồ rứa để giới hạn lại cho màn hình to k cái row nó dài quá